caiman/external/cell_magic_wand.py

```diff
@@ -22,8 +22,7 @@
 ###########################################################################
 
 import numpy as np
-from scipy.ndimage.interpolation import zoom
-from scipy.ndimage.morphology import binary_fill_holes
+from scipy.ndimage import zoom, binary_fill_holes
```

Comment:
This is fine; I would like to eventually stop using named imports in the codebase and just import libraries, using paths to functions, but there's no need to adjust that in this diff. If I were not trying to do that I'd suggest keeping function names in imports like this in alphabetical order, but in the long run I hope we'll get away from this style of import

Reply:
I think named imports are fine in modules that specially deal with just a few of them for a specific purpose.

Reply:
I don't think they're terrible, but I think it mildly improves code to just use the full name to something. When you read scipy.ndimage.zoom() it's very clear where it's from. zoom() is a little bit less clear.

Reply:
true, I see that with something ambiguous like zoom